src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/components/TradeQuoteBadges.tsx

```diff
@@ -57,6 +79,24 @@ export const TradeQuoteBadges: React.FC<TradeQuoteBadgesProps> = ({
 
   return (
     <Flex alignItems='center' gap={2} {...rest}>
+      {isBoost && (
+        <QuoteBadge
+          icon={TbRocket}
+          color='purple.500'
```

Comment:
<img width="903" height="631" alt="image" src="https://github.com/user-attachments/assets/df49be60-5ba1-4831-8f4b-32621cdcc8fa" />

Not sure about this one but hey, that's in the Figma - happy to keep it the same green.500 as before for consistency if that's weird tbh, which I *do* believe it is 🤔